dspace-xmlui/src/main/java/org/dspace/app/xmlui/aspect/artifactbrowser/ItemViewer.java

```diff
@@ -292,7 +294,7 @@ else if (doiMatcher.lookingAt())
         if (xHTMLHeadCrosswalk == null)
         {
             xHTMLHeadCrosswalk = (DisseminationCrosswalk) PluginManager.getNamedPlugin(
```

Comment:
PluginManager is replaced with CoreServiceFactory.getInstance().getPluginService()